from unittest import TestCase


class TestMain(TestCase):
    def test_main(self):
        self.fail()
